### PR TITLE
add imagescaler support to backend

### PR DESCRIPTION
**Pull request by weikexin**
Scale and bias the input image.
In the onnx difinition:
* Input tensor has shape [N,C,H,W].
* Output has same shape and type as input.
* Bias applied to each channel, same size as C.

### CODE REVIEW

File: onnx_tf/handlers/backend/image_scaler.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+@onnx_op("ImageScaler")
+class ImageScaler(BackendHandler):
+
+  @classmethod
+  def version_1(cls, node, **kwargs):
+    input_dict = kwargs["tensor_dict"]
+    x = input_dict[node.inputs[0]]
+    scale = node.attrs["scale"]
```
Comment (fumihwh) [inline onnx_tf/handlers/backend/image_scaler.py]: `scale` is not a required attr and has default value 1.0f. 
Should use `node.attrs.get("scale", 1.0)`.

File: test/backend/test_node.py [hunk 2]
```diff
@@ -391,6 +391,25 @@ def test_global_average_pool(self):
...
+    # Bias: applied to each channel, same size as C
+    # Output has same shape and type as input
+    x = self._get_rnd([1, 3, 224, 224])
+    #random distribution over [0,1), so add 0.1
+    scale = np.random.rand(1)[0] + 0.1
+    bias = np.random.rand(3)
+    node_def = helper.make_node("ImageScaler", ["X"], ["Y"], scale=scale, bias=bias)
+    output = run_node(node_def, [x])
+    test_out = np.multiply(x, scale)
+    test_out = np.transpose(test_out, [0,2,3,1])
```
Comment (fumihwh) [inline test/backend/test_node.py]: Maybe not formatted.
Could you fix this?

Comment (weikexin) [reply test/backend/test_node.py]: OK

File: onnx_tf/handlers/backend/image_scaler.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  @classmethod
+  def version_1(cls, node, **kwargs):
+    input_dict = kwargs["tensor_dict"]
+    x = input_dict[node.inputs[0]]
+    scale = node.attrs.get("scale", 1.0)
+    output = tf.multiply(x, scale)
+    if "bias" in node.attrs:
+        bias = node.attrs["bias"]
+        output = tf.nn.bias_add(output, bias, data_format="NCHW")
```
Comment (fumihwh) [inline onnx_tf/handlers/backend/image_scaler.py]: Please use 2 spaces indent.
https://github.com/google/yapf
with `--style="{based_on_style: google, indent_width: 2}"`

Comment (weikexin) [reply onnx_tf/handlers/backend/image_scaler.py]: Sorry, I did not notice it, fix this now

Comment (fumihwh) [reply onnx_tf/handlers/backend/image_scaler.py]: @weikexin Thanks! Also `test_node.py` right?

Comment (weikexin) [reply onnx_tf/handlers/backend/image_scaler.py]: Yes.  `test_node.py` uses 2 spaces indent.

Comment (weikexin) [reply onnx_tf/handlers/backend/image_scaler.py]: OK

**Comment by fumihwh** [comment]
Thanks! Looks great!

**Pull request merged**